packages/sensors_plus/sensors_plus/android/src/main/kotlin/dev/fluttercommunity/plus/sensors/SensorsPlugin.kt

```diff
@@ -73,9 +73,12 @@ class SensorsPlugin : FlutterPlugin {
     }
 
     companion object {
-        private const val ACCELEROMETER_CHANNEL_NAME = "dev.fluttercommunity.plus/sensors/accelerometer"
+        private const val ACCELEROMETER_CHANNEL_NAME =
```

Comment:
Are you sure that your IDE doen't override our editorconfig file with formatting style in the root of the repo?

Reply:
Nevermind, all good.